apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -560,3 +554,77 @@ function wpcom_is_previewing_global_styles( ?int $user_id = null ) {
 	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
 	return ! isset( $_GET['hide-global-styles'] ) && user_can( $user_id, 'administrator' );
 }
+
+/**
+ * Checks whether the site has access to Global Styles with a Personal plan as part of an A/B test.
+ *
+ * @param  int $blog_id Blog ID.
+ * @return bool Whether the site has access to Global Styles with a Personal plan.
+ */
+function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) {
+	$owner_id = wpcom_get_blog_owner( $blog_id );
+	if ( ! $owner_id ) {
+		return false;
+	}
+
+	$owner = get_userdata( $owner_id );
+	if ( ! $owner ) {
+		return false;
+	}
```

Comment:
We can include these checks so that they only get executed if we need to add the blog sticker to the blog.